src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx

```diff
@@ -75,7 +75,10 @@ const OperatorDashboard = () => {
     );
   });
 
-  const openSingleOperator = (listIndex: number) => {
+  const openSingleOperator = (listIndex: number, e: any) => {
+    if ( e.target === e.currentTarget.querySelector('.openExplorer') ){
```

Comment:
please remove direct access to dom and use Ref 
https://legacy.reactjs.org/docs/refs-and-the-dom.html

Reply:
done